code/datums/candidate_poll.dm

```diff
@@ -138,8 +138,9 @@
 /datum/candidate_poll/proc/announce_chosen(list/poll_recipients)
 	if(!length(chosen_candidates))
 		return
-	for(var/mob/poll_recipient as anything in poll_recipients)
-		for(var/mob/chosen as anything in chosen_candidates)
-			if(isnull(chosen))
-				continue
-			to_chat(poll_recipient, span_ooc("[isobserver(poll_recipient) ? FOLLOW_LINK(poll_recipient, chosen) : null][span_warning(" [full_capitalize(role)] Poll: ")][key_name(chosen, include_name = FALSE)] was selected."))
+	for(var/mob/chosen as anything in chosen_candidates)
+		if(isnull(chosen))
+			continue
```

Comment:
```suggestion
	for(var/mob/chosen in chosen_candidates)
```

no need to `as anything` AND have a typecheck if we can just use the typecheck built into the for-loop